packages/webextension-v3/src/action/action.js

```diff
@@ -45,19 +45,23 @@ const login = async () => {
     chrome.storage.local.set({ token }, () => {
       chrome.storage.local.get(["seenTutorial"], (result) => {
         if (result.seenTutorial) {
-          document.getElementById("message").innerText =
-            "You are now logged in. Click the RecipeSage icon again to clip this website.";
+          let message = document.getElementById("message");
+          message.style.color = "green";
+          message.innerText =
+            "\nYou are now logged in. Click the RecipeSage icon again to clip\
```

Comment:
Any way we can avoid the leading newline here? Perhaps by having the message container reserve additional height via css?